src/rtty.c

```diff
@@ -202,6 +239,9 @@ int rx_rtty()
 	    ry_term[2][0] = '\0';
 	    ry_term[3][0] = '\0';
 	    ry_term[4][0] = '\0';
+	    /*for(i=0; i<13; i++) {
+		qtc_ry_lines[i][0] = '\0';
+	    }*/
```

Comment:
Do you need that piece of code? Please delete it or write a comment why it is here and when the comment should be dropped.